py3status/modules/yubikey.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+"""
+Show an indicator when YubiKey is waiting for a touch.
+
+Configuration parameters:
+    cache_timeout: How often to detect a pending touch request.
+        (default 1)
+    format: Display format for the module.
+        (default '\?if=is_waiting&color=bad Y')
+    u2f_keys_path: Full path to u2f_keys if you want to monitor sudo access.
+        (default '~/.config/Yubico/u2f_keys')
+
+SAMPLE OUTPUT
+{'color': '#FF0000', 'full_text': 'Y'}
```

Comment:
I think I'd like something more readable like...
* `Waiting`
* `Key Waiting`
* `YubiKey`
* `Key` <-- @lasers's pick 
* `KEY`
* `KEY!`

Reply:
Yeah, I'm personally always using some fancy icons, so coming up with a default text message was a tough choice...

Not to confuse what key is the key that is waiting, how about a more specific `Touch YubiKey`?

![image](https://user-images.githubusercontent.com/1177900/31578531-57115728-b123-11e7-8ff5-aad5ac5e6ead.png)

Reply:
I feel it's a bit on the nose and I'd like to stay with with subtle approach for QA purposes. Maybe just `YubiKey` or `Key` as I believe people will pick this up quicker than our expectations... And I feel that seeing `Touch YubiKey` repeatedly could be tiresome. It is akin to seeing `Check Gmail` repeatedly too... when `Gmail` or `Mail` would work just fine here.

Same thing with `key_path`. Make `yubikey` look clean in the config. (All my QA opinions).

Reply:
`YubiKey` it is then. Config name `u2f_keys_path` will leave for the reasons described in another comment.